new-test/app.py

```
@@ -0,0 +1,129 @@
+import logging
+import threading
+import socket
+
+app = Flask(__name__)
+
+# Security Issue: Hardcoded credentials
+DB_PASSWORD = "super_secret_password123"
+JWT_SECRET = "my_jwt_secret_key"
+
+# Security Issue: Insecure database configuration
+app.config['SQLALCHEMY_DATABASE_URI'] = f'sqlite:///app.db'
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)